src/screens/FeedScreen/RenderList.js

```diff
@@ -211,15 +232,19 @@ const styles = StyleSheet.create({
   footerWrapper: (h) => ({height: h, alignItems: 'center', justifyContent: 'center'}),
   contentReaction: (heightReaction) => ({
     maxHeight: heightReaction,
-    marginBottom: heightReaction <= 0 ? tabBarHeight + 10 : 0
+    marginBottom: heightReaction <= 0 ? tabBarHeight + normalize(10) : 0
   }),
   cardContainer: {
     width: FULL_WIDTH,
     borderBottomWidth: 0,
     borderBottomColor: COLORS.lightgrey,
     backgroundColor: 'white',
     height: dimen.size.FEED_CURRENT_ITEM_HEIGHT,
-    marginBottom: normalizeFontSizeByWidth(4)
+    marginBottom: normalizeFontSizeByWidth(4),
+    shadowColor: COLORS.black000,
+    shadowOffset: {width: 0, height: 4},
+    shadowOpacity: 0.3,
+    shadowRadius: 4
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

The `normalize` function is removed from the marginBottom calculation in the `contentReaction` style. This could potentially cause layout issues if the `tabBarHeight` is not already normalized. If normalization is necessary, consider adding it back.
```diff
- marginBottom: heightReaction <= 0 ? tabBarHeight + normalize(10) : 0
+ marginBottom: heightReaction <= 0 ? tabBarHeight + 10 : 0
```



<!-- This is an auto-generated comment by OSS CodeRabbit -->